collision_detection_fcl/src/collision_common.cpp

```diff
@@ -455,6 +455,133 @@ bool distanceCallback(fcl::CollisionObject* o1, fcl::CollisionObject* o2, void*
   return cdata->done_;
 }
 
+
+bool distanceDetailedCallback(fcl::CollisionObject* o1, fcl::CollisionObject* o2, void* data, double& min_dist)
```

Comment:
They are currently not being checked within the callback function. I believe it would be unlikely for the callback function to receive a null pointer, unless someone called it directly instead of passing the callback function to the FCLManager::distance function. I would need to explore FCL further to determine if FCLMangers would every pass a null pointer.